docs/docs/contributions/releases/release-process.mdx

```diff
@@ -151,7 +125,7 @@ Run this script as a basic smoke test:
 
 You should also check [GitHub Releases](https://github.com/pantsbuild/pants/releases) to ensure everything looks good. Find the version you released, then click it and confirm that the "Assets" list includes PEXes for macOS and Linux.
 
-## Step 5: Run release testing on public repositories
+## Step 4: Run release testing on public repositories
```

Comment:
I love the reduction in number of steps. Let's get that down to 1!!

Reply:
[You were saying?](https://github.com/pantsbuild/pants/pull/20438/files#diff-576fbbbd0d8f6c9919a98c10ec11653dca55bcd4270a985e582af1716321d810) (Down to 4 steps :stuck_out_tongue: )